docs/docs/configuration.md

```diff
@@ -49,6 +52,70 @@ The following `Mirror` setting configures BaGetter to index packages from [nuget
 
 :::
 
+### Mirror Authentication
+
+<Tabs>
+  <TabItem value="None" label="None" default>
+    If your mirror does not require authentication, leave the `Mirror` configuration as is and you're good to go.
```

Comment:
You could also replace the current mirror section with the tab logic where the current "Enable read-through caching" section is the "No Authentication" tab content. Then the new "Enable read-through caching" section will have the tabs and the current content is automatically displayed as the first tab and users can switch to "add" authentication to the configuration.

Reply:
I meant just replacing the current paragraph with the new tab logic, not an entirely new page😅. The tabs aren't really bigger than the current section, so I think it's ok to keep it on the same page. And I'm not sure if it makes sense to change everything to primarily be called "mirror", as this doesn't actually mirror the upstream but really is just read-through caching. BaGetter doesn't download anything from upstream unless it is specifically requested.

Reply:
I tried to put FroggieFrog's idea into it, but ofc we can easily revert that. Trying variations and see what sticks is good.